cluster-provision/k8s/1.24/k8s_provision.sh

```diff
@@ -127,9 +132,12 @@ rules:
 EOF
 
 kubeadm_manifest="/etc/kubernetes/kubeadm.conf"
-envsubst < /tmp/kubeadm.conf > $kubeadm_manifest
+envsubst < $kubeadm_raw > $kubeadm_manifest
+
+until ip address show dev eth0 | grep global | grep inet6; do sleep 1; done
```

Comment:
Is there a case here that this will sleep continuously if the interface fails to get an IPv6 address? Should it fail at some point?

Reply:
well if the dnsmasq has a bug it theoretically might, but we didn't see it never yet (we are already using this logic for few months now)
In case we see we can add a protection, just so it will break after few minutes, imo there is no need now unless you think otherwise